test/NosCore.Tests/Helpers/TestHelpers.cs

```diff
@@ -261,26 +261,27 @@ public ClientSession GenerateSession()
         {
             _lastId++;
             var acc = new AccountDto
-                {AccountId = _lastId, Name = "AccountTest" + _lastId, Password = "test".ToSha512()};
+            { AccountId = _lastId, Name = "AccountTest" + _lastId, Password = "test".ToSha512() };
             AccountDao.InsertOrUpdate(ref acc);
             var minilandProvider = new Mock<IMinilandProvider>();
-            var session = new ClientSession(WorldConfiguration, MapInstanceProvider, null, _logger,
+            var session = new ClientSession(WorldConfiguration, MapInstanceProvider, new Mock<IExchangeProvider>().Object, _logger,
                 new List<IPacketHandler>
                 {
                     new CharNewPacketHandler(CharacterDao, MinilandDao),
                     new BlInsPackettHandler(BlacklistHttpClient.Object),
                     new UseItemPacketHandler(),
                     new FinsPacketHandler(FriendHttpClient.Object, ChannelHttpClient.Object,
                         ConnectedAccountHttpClient.Object),
-                    new SelectPacketHandler(CharacterDao, _logger, null, MapInstanceProvider,
-                        _itemInstanceDao, _inventoryItemInstanceDao, _staticBonusDao, null, null)
-                }, FriendHttpClient.Object, null, PacketHttpClient.Object, minilandProvider.Object)
+                    new SelectPacketHandler(CharacterDao, _logger, new Mock<IItemProvider>().Object, MapInstanceProvider,
+                        _itemInstanceDao, _inventoryItemInstanceDao, _staticBonusDao, new Mock<IGenericDao<QuicklistEntryDto>>().Object, new Mock<IGenericDao<TitleDto>>().Object)
+                }, FriendHttpClient.Object, new Mock<ISerializer>().Object, PacketHttpClient.Object, minilandProvider.Object)
             {
                 SessionId = _lastId
             };
+
             var chara = new GameObject.Character(new InventoryService(ItemList, session.WorldConfiguration, _logger),
-                new ExchangeProvider(null, WorldConfiguration, _logger), null, CharacterDao, null, null, AccountDao,
-                _logger, null, null, null, null, null)
+                new ExchangeProvider(new Mock<IItemProvider>().Object, WorldConfiguration, _logger), new Mock<IItemProvider>().Object, CharacterDao, new Mock<IGenericDao<IItemInstanceDto>>().Object, new Mock<IGenericDao<InventoryItemInstanceDto>>().Object, AccountDao,
+                _logger, new Mock<IGenericDao<StaticBonusDto>>().Object, new Mock<IGenericDao<QuicklistEntryDto>>().Object, new Mock<IGenericDao<MinilandDto>>().Object, minilandProvider.Object, new Mock<IGenericDao<TitleDto>>().Object)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 235 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)